packages/cli/package.json

```diff
@@ -64,21 +76,21 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@babel/preset-env": "^7.3.4",
     "@preact/async-loader": "^3.0.0-next.16",
-    "autoprefixer": "^9.4.9",
+    "autoprefixer": "^9.4.10",
     "babel-esm-plugin": "^0.4.0",
     "babel-loader": "^8.0.5",
     "babel-plugin-macros": "^2.4.5",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "console-clear": "^1.0.0",
-    "copy-webpack-plugin": "^5.0.0",
+    "copy-webpack-plugin": "^5.0.1",
     "critters-webpack-plugin": "^1.3.3",
     "cross-spawn-promise": "^0.10.1",
-    "css-loader": "^2.1.0",
+    "css-loader": "^2.1.1",
     "ejs-loader": "^0.3.0",
-    "esm": "^3.2.7",
+    "esm": "<=3.2.14",
```

Comment:
https://github.com/standard-things/esm/issues/747

Reply:
`esm` [v3.2.17](https://github.com/standard-things/esm/releases/tag/3.2.17) is released :tada: